### PR TITLE
Refactoring code to adjust to Sonar policies

### PR DESCRIPTION
**Pull request by neilprado**

### CODE REVIEW

File: heimdall-api/src/main/java/br/com/conductor/heimdall/api/configuration/SwaggerConfiguration.java [hunk 1]
```diff
@@ -49,38 +49,36 @@
...
-
-          return docket;
+         
+          return new Docket(DocumentationType.SWAGGER_2)
+                  .select()
+                  .apis(RequestHandlerSelectors.basePackage("br.com.conductor.heimdall.api.resource"))
+                  .paths(PathSelectors.any())
+                  .build()
+                  .apiInfo(apiInfo())
+                  .securitySchemes(Collections.singletonList(new ApiKey(AUTHORIZATION, AUTHORIZATION, "header")))
```
Comment (smrafael) [inline heimdall-api/src/main/java/br/com/conductor/heimdall/api/configuration/SwaggerConfiguration.java:68]: Make string `header` as constant too

File: heimdall-core/src/main/java/br/com/conductor/heimdall/core/repository/jdbc/InterceptorJDBCRepository.java [hunk 2]
```diff
@@ -36,23 +36,26 @@
 
     private JdbcTemplate jdbcTemplate;
 
+    private static final String SQL = "SELECT I.ID, I.API_ID, I.EXECUTION_POINT, I.LIFE_CYCLE, I.TYPE, I.NAME, I.CONTENT, I.EXECUTION_ORDER, I.STATUS, " +
```
Comment (smrafael) [inline heimdall-core/src/main/java/br/com/conductor/heimdall/core/repository/jdbc/InterceptorJDBCRepository.java:39]: SQL is not a good name for a variable. Change to a name that makes more sense

File: heimdall-api/src/main/java/br/com/conductor/heimdall/api/configuration/SwaggerConfiguration.java (hunk 1 above)
Comment (neilprado) [reply heimdall-api/src/main/java/br/com/conductor/heimdall/api/configuration/SwaggerConfiguration.java]: done

File: heimdall-core/src/main/java/br/com/conductor/heimdall/core/repository/jdbc/InterceptorJDBCRepository.java (hunk 2 above)
Comment (neilprado) [reply heimdall-core/src/main/java/br/com/conductor/heimdall/core/repository/jdbc/InterceptorJDBCRepository.java]: Changed to LIST_INTERCEPTOR

File: heimdall-core/src/main/java/br/com/conductor/heimdall/core/util/ConstantsPath.java [hunk 3]
```diff
@@ -37,6 +37,8 @@ private ConstantsPath() { }
      
      public static final String PATH_API = "/v1/api";
 
+     public static final String APIID = "/{apiId}";
```
Comment (smrafael) [inline heimdall-core/src/main/java/br/com/conductor/heimdall/core/util/ConstantsPath.java:40]: Change constant name to `API_ID_URL_PATH_VAR`

Comment (neilprado) [reply heimdall-core/src/main/java/br/com/conductor/heimdall/core/util/ConstantsPath.java]: Done

**Pull request merged**